dataland-frontend/tests/component/components/pages/LandingPage.cy.ts

```diff
@@ -0,0 +1,126 @@
+import { checkButton, checkImage, checkLinkByContent, checkLinkByTarget } from "@ct/testUtils/ExistenceChecks";
+import NewLandingPage from "@/components/pages/NewLandingPage.vue";
+import { minimalKeycloakMock } from "@ct/testUtils/Keycloak";
+
+describe("Component test for the landing page", () => {
+  it("Check if essential elements are present", () => {
+    cy.mountWithPlugins(NewLandingPage, {
+      keycloak: minimalKeycloakMock({
+        authenticated: false,
+      }),
+    }).then(() => {
+      validateTopBar();
+      validateIntroSection();
+      validateBrandsSection();
+      // TODO test I AM INTERESTED button
+      // TODO test get in contact button
+
+      validateQuotesSlides();
+      validateHowItWorksSlides();
+      checkNewFooter();
+    });
+  });
+});
+
+/**
+ * Validates the elements of the top bar
+ */
+function validateTopBar(): void {
+  /**
+   * Gets the top bar element
+   * @returns the top bar element
+   */
+  function getTopBar(): Cypress.Chainable {
+    return cy.get("header");
+  }
+  checkImage("Dataland banner logo", "gfx_logo_dataland_orange_S.svg", getTopBar());
+  checkButton("signup_dataland_button", "Sign Up", getTopBar());
+  checkLinkByContent("Login", getTopBar());
+}
+
+/**
+ * Validates the elements of the intro section
+ */
+function validateIntroSection(): void {
+  checkImage("Liberate Data -  Empower Autonomy. Dataland, the Open ESG Data Platform.", "gfx_logo_d_orange_S.svg");
+  cy.get("h1").should("contain.text", "Liberate Data");
+  cy.get("h1").should("contain.text", "Empower Autonomy");
+}
+
+/**
+ * Validates the images of the brands trusting in dataland
+ */
+function validateBrandsSection(): void {
+  checkImage("Brand 1", "img_chom_capital.png");
+  checkImage("Brand 2", "img_envoria.png");
+  checkImage("Brand 3", "img_fuerstl_bank.png");
+  checkImage("Brand 4", "img_ampega.png");
+  checkImage("Brand 5", "img_metzler.png");
+  checkImage("Brand 6", "img_deka.png");
```

Comment:
Probably also possible to dynamically get the values from the content.json

Reply:
Lets discuss this.

Reply:
Done